.github/workflows/release.yml

```diff
@@ -73,26 +73,34 @@ jobs:
     uses: ./.github/workflows/assemble-xcframework-variant.yml
     secrets: inherit
     with:
-      name: ${{matrix.variant.name}}
+      scheme: ${{matrix.variant.scheme}}
       suffix: ${{matrix.variant.suffix}}
       configuration-suffix: ${{matrix.variant.configuration-suffix}}
       variant-id: ${{matrix.variant.id}}
       signed: true
       release-version: ${{ github.event.inputs.version }}
+      excluded-archs: ${{matrix.variant.excluded-archs}}
+      override-name: ${{matrix.variant.override-name}}
     strategy:
       matrix:
         variant:
-          - name: Sentry
+          - scheme: Sentry
             macho-type: mh_dylib
             suffix: "-Dynamic"
             id: sentry-dynamic
-          - name: Sentry
+            excluded-archs: arm64e
```

Comment:
`h`: Do we have documentation somewhere why we remove the arm64e from the default `Sentry.xcframework` and only add it in `Sentry-Dynamic-WithARM64e`? This seems like a major breaking change.

Reply:
Note added